### PR TITLE
Feature 437 tc rmw

### PR DESCRIPTION
**Pull request by georgemccabe**
A new wrapper (TCRMW) and 3 new use cases are found in this PR. I also added new tests for config and the new wrapper. The new cases can be run on kiowa.

### CODE REVIEW

File: docs/Users_Guide/wrappers.rst [hunk 1]
```diff
@@ -1483,7 +1483,296 @@ Configuration
...
+| :term:`TC_RMW_REGRID_METHOD`
+| :term:`TC_RMW_REGRID_WIDTH`
+| :term:`TC_RMW_REGRID_VLD_THRESH`
+| :term:`TC_RMW_REGRID_SHAPE`
+| :term:`TC_RMW_N_RANGE`
+| :term:`TC_RMW_N_AZIMUTH`
+| :term:`TC_RMW_MAX_RANGE_KM`
+| :term:`TC_RMW_DELTA_RANGE_KM`
+| :term:`TC_RMW_SCALE`
+| :term:`TC_RMW_STORM_ID`
```
Comment (JohnHalleyGotway) [inline docs/Users_Guide/wrappers.rst:1527]: Shouldn't we have an entry here for "TC_RMW_MODEL"?

We use a generic 'MODEL' variable that applies to many wrappers. Do you think we need a wrapper-specific variable for this?

File: docs/Users_Guide/wrappers.rst [hunk 2]
```diff
@@ -1483,7 +1483,296 @@ Configuration
...
+| :term:`TC_RMW_BASIN`
+| :term:`TC_RMW_CYCLONE`
+| :term:`TC_RMW_STORM_NAME`
+| :term:`TC_RMW_INIT_INCLUDE`
+| :term:`TC_RMW_VALID_BEG`
+| :term:`TC_RMW_VALID_END`
+| :term:`TC_RMW_VALID_INCLUDE_LIST`
+| :term:`TC_RMW_VALID_EXCLUDE_LIST`
+| :term:`TC_RMW_VALID_HOUR_LIST`
+
```
Comment (JohnHalleyGotway) [inline docs/Users_Guide/wrappers.rst]: Should also have an entry for TC_RMW_LEAD_LIST.

George: The wrapper uses LEAD_SEQ to build the lead list passed into the MET config file. It is the same variable used to loop over forecast leads in other wrappers. I think it makes sense to use that same variable to be consistent with other wrappers.

File: docs/Users_Guide/wrappers.rst [hunk 3]
```diff
@@ -1483,7 +1483,296 @@ Configuration
...
+MET Configuration
+~~~~~~~~~~~~~~~~~
+
+This is the MET configuration file used for this wrapper. Below the file contents are descriptions of each environment variable referenced in this file and how the METplus configuration variables relate to them.
+
+.. literalinclude:: ../../../parm/met_config/TCRMWConfig_wrapped
+
+The following environment variables are referenced in the MET configuration file. The values are generated based on values in the METplus configuration files.
+
```
Comment (JohnHalleyGotway) [inline docs/Users_Guide/wrappers.rst:1548]: Add entry for model here.

**Review by JohnHalleyGotway — CHANGES_REQUESTED** [review]
Please make minor updates to the documentation, as noted in single comments.

**Review by JohnHalleyGotway — APPROVED** [review]
I approve of these changes.

**Comment by georgemccabe** [comment]
Latest changes only involve documentation and previous test runs in Travis succeeded, so merging into develop even though travis has not completed.

**Pull request merged**